static/js/ui.js

```diff
@@ -13,34 +15,37 @@ exports.replace_emoji_with_text = function (element) {
     });
 };
 
-exports.set_up_scrollbar = function (element) {
-    var perfectScrollbar = new PerfectScrollbar(element[0], {
-        suppressScrollX: true,
-        useKeyboard: false,
-        wheelSpeed: 0.68,
-        scrollingThreshold: 50,
-        minScrollbarLength: 40,
-    });
-    element[0].perfectScrollbar = perfectScrollbar;
+exports.set_up_scrollbar = function (element_selector) {
+    new SimpleBar(element_selector.expectOne()[0]);
+};
```

Comment:
We should consider preserving the settings we had before.  From https://github.com/Grsmto/simplebar#options, should we:

* Set `scrollbarMinSize` should be 40 to match `minScrollbarLength` above?
* Include an option to `set_up_scrollbar` to set `forceVisible`?  (I think we have at least one place we'll want to use this).
* Do something to emulate the `suppressScrollX` feature (https://github.com/Grsmto/simplebar#apply-scroll-vertically-only)?  I guess mostly we'll need to find the places that need this in QA.

Reply:
(The place we wanted forceVisible was on the right panel of "settings")

Reply:
* The SimpleBar default `scrollbarMinSize = 25` seems fine to me. (perfect-scrollbar had no default `minScrollbarLength`.)
* `forceVisible` is easy to add later, so let’s do one thing at a time.
* We don’t need to do anything for `suppressScrollX` if the CSS is correct, so yeah, let’s make the CSS correct if it isn’t already.

Reply:
ok